### PR TITLE
Added my inefficient java solution

### PR DESCRIPTION
**Pull request by Aniketsaoji**

### CODE REVIEW

File: week2/aniketsaoji/fizzbuzz.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		Random rand = new Random();
+		return rand.nextInt(100) + 1;	
+	}
+
+	public static void main (String [] args) {
+		int count = 1;
+		int randomNumber = returnRandom();
+		while (count < 101) {
+			while (randomNumber != count) {
+				randomNumber = returnRandom();
```
Comment (jayelm) [inline week2/aniketsaoji/fizzbuzz.java:15]: This should be deterministic, i.e. should always print out numbers 1-100, not random numbers

Comment (Aniketsaoji) [inline week2/aniketsaoji/fizzbuzz.java:15]: It does. Run the code

Comment (jayelm) [inline week2/aniketsaoji/fizzbuzz.java:15]: Oh damn, didn't notice what the while loop was doing. You weren't joking about an inefficient solution

File: week2/aniketsaoji/fizzbuzz.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			}
+			if ((randomNumber%3 == 0) && (randomNumber%5 == 0))
```
Comment (rootulp) [inline week2/aniketsaoji/fizzbuzz.java:17]: You might want to include [horizontal whitespace](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace) around `%` for readability:
`if ((randomNumber % 3 == 0) && (randomNumber % 5 == 0))`

**Pull request merged**